docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -162,21 +174,14 @@ Preparing For a Release
 #. Ensure all the required tests pass on BuildBot:
 
    Pushing the branch in the previous step should have started a build on BuildBot.
+   If not, you can force a build by logging in to BuildBot, entering the release branch name in to the box at the top right and clicking the ``Force`` button.
 
-   Unfortunately it is acceptable or expected for some tests to fail.
-   Discuss with the team whether the release can continue given any failed tests.
+   Build failures in the ``Expected failures`` section may be safely ignored.
```

Comment:
I think that this is expected, the build failures don't show up on GitHub, so this sentence can be removed.